recipes/openssh/all/conanfile.py

```diff
@@ -0,0 +1,132 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import patch, copy, get, rmdir, export_conandata_patches
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps
+from conan.tools.layout import basic_layout
+from os.path import join
+
+required_conan_version = ">=1.54.0"
+
+
+class PackageConan(ConanFile):
+    name = "openssh"
+    description = "The OpenSSH (portable) suite of secure connectivity tools"
+    license = "BSD"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/openssh/openssh-portable"
+    topics = ("security", "cryptography", "login", "keychain", "file-sharing")
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_openssl": [True, False],
```

Comment:
Does the project support using `libressl` as an alternative?

```suggestion
        "with_libcrypto": [None, "libressl", "openssl],
```

Reply:
No it does not seem to have config options for that!

Reply:
I think it should still work with `libressl` without a config option. The `libressl` package should work since it should be a drop-in replacement.

Reply:
Added support for libressl